target/linux/ramips/image/mt7620.mk

```diff
@@ -32,6 +32,28 @@ define Device/alfa-network_ac1200rm
 endef
 TARGET_DEVICES += alfa-network_ac1200rm
 
+define Device/ApexeraNetBot
+  DTS := ApexeraNetBot
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
```

Comment:
ralink_default_fw_size_16M == 16121856 bytes but your firmware partition is 0xfb0000 bytes == 16449536 bytes.

Use `IMAGE_SIZE := 16064k`